test/e2e/tls_test.go

```diff
@@ -225,8 +225,33 @@ func TestTLSEnabledWithNamedCert(t *testing.T) {
 	)
 
 	// wait until pods are in Ready status
-	k8s.WaitUntilPodAvailable(t, kubectlOptions, podName, 15, 30*time.Second)
-	k8s.WaitUntilPodAvailable(t, kubectlOptions, podOneName, 15, 30*time.Second)
+	k8s.RunKubectl(t, kubectlOptions, "get", "pods")
+	k8s.RunKubectl(t, kubectlOptions, "get", "ns")
```

Comment:
The output of these get commands is unused further in the tests. Is there a specific reason to get ns, sts, svc, and PVC here?
If it's required for the test verification, please add comments for these checks.

Reply:
I am doing debug trying to find out the issue.